pkg/reconciler/route/traffic/rollout.go

```diff
@@ -70,3 +72,94 @@ type RevisionRollout struct {
 	// target percentage.
 	Percent int `json:"percent"`
 }
+
+// Roll processes previous Rollout object and compares to the current
+// rollout state. If there is different, Merge will start or stop the rollout
+// and update `RevisionRollout` objects accoringly.
+// Roll returns true if any changes have been made.
```

Comment:
changes have been made to what?  `cur` or `prev`?

Reply:
Added the comment.